### PR TITLE
MDI content transitions

### PR DESCRIPTION
**Pull request by rpassas**
## Summary

Added MDI content transitioning or worth highlighting from our harmonization proposal. 

Omitted: bundles, compositions, messages

### CODE REVIEW

File: input/pagecontent/mdi-content-transitions.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+|Death Date |  MDI Profiles  |    -    | [ObservationDeathDate] |  [ObservationDeathDateVitalRecordsNew]  |   -      |
+|How Death Injury Occurred | MDI Profiles |  -  | [ObservationHowDeathInjuryOccurred] | [ObservationInjuryIncidentVitalRecordsNew]  | - |
+|Manner of Death |  MDI Profiles  |    -    | [ObservationMannerOfDeath] | [ObservationMannerOfDeathVitalRecordsNew] |   -      |
+|Decedent Pregnancy |  MDI Profiles  |    -    | [ObservationDecedentPregnancy] | [ObservationDecedentPregnancyVitalRecordsNew] | - |
+|Tobacco Use Contributed to Death |  MDI Profiles  | - | [ObservationTobaccoUseContributedToDeath] |    [ObservationTobaccoUseContributedToDeathVitalRecordsNew] | - |
+|Autopsy Performed Indicator | MDI Profiles  |    -    | [ObservationAutopsyPerformedIndicator] |    -    |   -   |
+|Death Certification |  MDI Profiles  |    -    | [ProcedureDeathCertification] | [ProcedureDeathCertificationVitalRecordsNew] |  - |
+|Death Location |  MDI Profiles  |    -    | [LocationDeath] |  [LocationDeathVitalRecordsNew]  |   -      |
+|Injury Location |  MDI Profiles  |    -    | [LocationInjury] | [LocationInjuryVitalRecordsNew] |   -      |
+{: .grid }
```
Comment (saulakravitz) [inline input/pagecontent/mdi-content-transitions.md:16]: Include the medical-history section, with reference to USCore Conditions.

File: input/pagecontent/mdi-content-transitions.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+| :-------------: | ----- | ------- | ------------------ | ------------ | :-----: | 
+|Cause of Death Part 1 |  MDI Profiles  |  -  | [ObservationCauseOfDeathPart1] | [ObservationCauseOfDeathPart1VitalRecordsNew] |   -   |
+|Cause of Death Part 2 |   MDI Profiles  |    -    | [ObservationContributingCauseOfDeathPart2] | [ObservationCauseOfDeathPart2VitalRecordsNew] | - |
+|Death Date |  MDI Profiles  |    -    | [ObservationDeathDate] |  [ObservationDeathDateVitalRecordsNew]  |   -      |
+|How Death Injury Occurred | MDI Profiles |  -  | [ObservationHowDeathInjuryOccurred] | [ObservationInjuryIncidentVitalRecordsNew]  | - |
+|Manner of Death |  MDI Profiles  |    -    | [ObservationMannerOfDeath] | [ObservationMannerOfDeathVitalRecordsNew] |   -      |
+|Decedent Pregnancy |  MDI Profiles  |    -    | [ObservationDecedentPregnancy] | [ObservationDecedentPregnancyVitalRecordsNew] | - |
+|Tobacco Use Contributed to Death |  MDI Profiles  | - | [ObservationTobaccoUseContributedToDeath] |    [ObservationTobaccoUseContributedToDeathVitalRecordsNew] | - |
+|Autopsy Performed Indicator | MDI Profiles  |    -    | [ObservationAutopsyPerformedIndicator] |    -    |   -   |
```
Comment (saulakravitz) [inline input/pagecontent/mdi-content-transitions.md:12]: Add autopsy location with reference to USCoreLocation.  We will probably add a location profile for this to make it easier to query, just like we did for death location, injury location, and disposition location.

File: input/pagecontent/mdi-content-transitions.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# MDI to EDRS
+
+| Name of Profile | Group | change  |  Current Profile   | New Profile  | Comment |
+| :-------------: | ----- | ------- | ------------------ | ------------ | :-----: | 
+|Cause of Death Part 1 |  MDI Profiles  |  -  | [ObservationCauseOfDeathPart1] | [ObservationCauseOfDeathPart1VitalRecordsNew] |   -   |
```
Comment (saulakravitz) [inline input/pagecontent/mdi-content-transitions.md:5]: Add Decedent, with reference to USCorePatient.   (subject field)
Add Medical Examiner/Coroner (Certifier), with reference to USCorePractitioner (attester field).

**Review by saulakravitz — CHANGES_REQUESTED** [review]

File: input/pagecontent/mdi-content-transitions.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# MDI to EDRS
```
[the remark this answers is not in the document]
Comment (rpassas) [reply input/pagecontent/mdi-content-transitions.md:1]: This was included, but I added "performer" as well.

**Review by saulakravitz — APPROVED** [review]

**Pull request merged**